Date: Mar 19, 2019 2:31 AM

From: f26304b302c33c476dd81a012819a81bc18db0ba Mar 21, 2019 4:24 AM

Subject: [PATCH] RDK-22703 : Added support for AmazonRemote

	Rename patch
	Disable	 Secondary services discovery Receiving additional events
	which prevent a successful connnection to Amazon Remote. Temporary change only.
	Will need to be enabled back later. - Scan and Pair/Connect to AmazonRemote.

Source: COMCAST

License: GPL-2.0-or-later

Upstream-Status: Pending

Signed-off-by: Chandresh Pitty <Chandresh_Pitty@cable.comcast.com>
---

Index: bluez-5.45/src/shared/gatt-client.c
===================================================================
--- bluez-5.45.orig/src/shared/gatt-client.c
+++ bluez-5.45/src/shared/gatt-client.c
@@ -1125,10 +1125,14 @@ secondary:
 	if (client->discovery_req)
 		return;
 
+#if 0
 	util_debug(client->debug_callback, client->debug_data,
 				"Failed to start secondary service discovery");
 	discovery_op_unref(op);
 	success = false;
+#else
+    return;
+#endif
 
 done:
 	discovery_op_complete(op, success, att_ecode);
